public/static/docs/command-reference/repro.md

```diff
@@ -45,6 +45,39 @@ files, intermediate or final results. It saves all the data files, intermediate
 or final results into the <abbr>DVC cache</abbr> (unless `--no-commit` option is
 specified), and updates stage files with the new checksum information.
 
+### Parallel stage execution
+
+Currently `dvc repro` is not able to parallelize execution by itself (see
+[iterative/dvc#755](https://github.com/iterative/dvc/issues/755)), so if you
+need to do that you could launch multiple `dvc repro`s yourself. For example,
+say your DAG looks something like:
+
+```
+$ dvc pipeline show --ascii result.py
++--------+          +--------+
+| A1.dvc |          | B1.dvc |
++--------+          +--------+
+     *                   *
+     *                   *
+     *                   *
++--------+          +--------+
+| A2.dvc |          | B2.dvc |
++--------+          +--------+
+          *         *
+           **     **
+             *   *
+        +------------+
+        | result.dvc |
+        +------------+
+```
+
+so it consists of two pipeline branches (pipeline `A` and pipeline `B`) and the
+final `result` stage. To reproduce both branches at the same time, you could run
+`dvc repro A2.dvc` and `dvc repro B2.dvc` at the same time (e.g. by running them
+in separate terminals). After both are done running, you could then run
+`dvc repro result.dvc` that will see that both branches are already up-to-date
+and will only run the final stage.
```

Comment:
This pipeline consists of two parallel branches (`A` and `B`), and the final "result" stage, where the branches merge. To reproduce both branches at the same time, you could run `dvc repro A2.dvc` and `dvc repro B2.dvc` at the same time (e.g. in separate terminals). After both finish successfully, you can then run `dvc repro result.dvc`: DVC will know that both branches are already up-to-date and only execute the final stage.